projects/smart-table/src/lib/smart-table/smart-table.component.html

```diff
@@ -109,7 +109,7 @@
                  [loading]="rowsLoading && !(error$ | async)" [columns]="visibleColumns$ | async"
                  [rows]="rows$ |  async"
                  [hasClickAction]="true"
-                 [activeSorting]="orderBy"
+                 [activeSorting]="orderBy | async"
```

Comment:
Can you maybe add a $ to the name of orderBy so that it's clear its an observable?